modules/core/src/system.cpp

```diff
@@ -660,6 +662,10 @@ struct HWFeatures
         have[CV_CPU_RVV] = true;
     #endif
 
+    #if defined __loongarch_asx
+        have[CV_CPU_LASX] = true;
+    #endif
```

Comment:
You are best to enable these SIMD features by reading the CPUID like ARM and x86.

Reply:
> You are best to enable these SIMD features by reading the CPUID like ARM and x86.

Ok，thanks very much.

Reply:
After I copied the first patch to my local opencv repository (Tag: 4.5.4),  the Calib3d_StereoBM.regression test failed.
Note that the cmake option I used on 3A5000 platform is " -DBUILD_PNG=ON -D WITH_OPENCL=OFF -DBUILD_WITH_DEBUG_INFO=ON -DCPU_BASELINE=LASX ".

Reply:
I‘m so sorry that I didn't get the message or emails. I'm very glad you have a 3a5000 environment as We are considering providing 3a5000 remote environment.

YES, the Calib3d_StereoBM.regression test is currently failed. We suspect that compiler optimization is the reason as DEBUG version is OK. We will follow up and fix the bug.

Reply:
@gititgo Do you have any news on the `Calib3d_StereoBM.regression` test failure?